tests/wdio/marked-spec.js

```diff
@@ -1,6 +1,7 @@
 Terra.describeViewports('md', ['huge'], () => {
   it('renders md', () => {
     browser.url('/raw/test/cerner-terra-dev-site/md');
+    browser.moveToObject('#root', 0, 900);
```

Comment:
```suggestion
    browser.moveToObject('#root', 0, 768);
```
So we can testing in firefox/IE without errors. (Can't remember which one errors out if the position is larger than the viewport height).